packages/dcos-integration-test/extra/test_rlimit.py

```diff
@@ -0,0 +1,27 @@
+import subprocess
+import uuid
+
+
+def test_if_rlimits_can_be_used(cluster):
+    '''This test verifies that rlimits can be used.
+
+    Since marathon does not support rlimits yet we use `mesos-execute` as
+    scheduler. We run a job for which we specify an unlimited `RLIMIT_CORE`.
+    The job then examines the actual limit, and returns a failure if another
+    value is found.'''
+
+    argv = [
+        '/opt/mesosphere/bin/mesos-execute',
+        '--rlimits={"rlimits": [{"type":"RLMT_CORE"}]}',
+        '--master=leader.mesos:5050',
+        '--name={}'.format(uuid.uuid4().hex),
+        '--command=ulimit -c | grep -q unlimited',
+        '--shell=true',
+        '--env={"LC_ALL":"C"}']
+
+    output = subprocess.check_output(
+        argv,
+        stderr=subprocess.STDOUT,
+        universal_newlines=True)
+
+    assert output.count('TASK_FINISHED')
```

Comment:
Do we know how to many times "TASK_FINISHED" comes? 

* (Simply doing `assert "TASK_FINISHED" in output` is equivalent to the above `.count`. 
* Doing a regex search for the pattern associated with TASK_FINISHED might be "tighter"